### DISCUSSION TITLE
Unable to access github on my computer for forgotten password and invalid adress

### DISCUSSION THREAD

**Discussion by Skavengers** [community/community#111267] [Other Feature Feedback, Questions, & Ideas]
Dear support or whoever read this,
I'm writing to request assistance with accessing my account. I am curently unable to access my account on my computer, and when I try to log in on my computer, I'm unable to reset my password as my email address  is considered incorrect or invalid.
Could you please help me resolbe the issue so that I can regain access to my account ? I would appreciate it if you could provide me on how to proceed.

Thank you for your time and assistance. I look forward to hearing back you soon.
Best regards![IMG_20240307_224201.jpg](https://github.com/community/community/assets/91607648/76bec55e-b229-4a07-a3dc-96127a7028e6)

**Comment by SebastienXia** [comment]
## Hi, @Skavengers 

> You can also open a ticket to a support [here](https://support.github.com/)

**Reply by Skavengers** [reply]
I would like to do it but I can't since it redirect me to the website where they ask  me to connect... But i dont know my password so I'm blocked.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Skavengers! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). You can also contact Support by writing them at support@github.com. Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!
